test/parallel/test-global-webcrypto.js

```diff
@@ -7,7 +7,7 @@ if (!common.hasCrypto)
 const assert = require('assert');
 const crypto = require('crypto');
 
-/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-restricted-properties, no-restricted-syntax */
```

Comment:
Do we still need to disable no-restricted-syntax?
```suggestion
/* eslint-disable no-restricted-properties */
```

Reply:
Looks like the `no-restricted-syntax` error comes from `test/.eslintrc.yaml` [here](https://github.com/nodejs/node/blob/main/test/.eslintrc.yaml#L53) I think the `no-restricted-properties` should also be added into `test/.eslintrc.yaml` in order to keep the rule consistent.

Reply:
Adding `no-restricted-properties` into `test/.eslintrc.yaml` will result in more linting errors in other test files. I have added a `TODO` which I will try to fix in a separate PR.